WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -346,6 +346,19 @@ extension OrderDetailsViewModel {
         ServiceLocator.stores.dispatch(action)
     }
 
+    func syncShippingLabels(onCompletion: ((Error?) -> ())? = nil) {
+        let action = ShippingLabelAction.synchronizeShippingLabels(siteID: order.siteID, orderID: order.orderID) { result in
+            switch result {
+            case .failure(let error):
+                DDLogError("⛔️ Error synchronizing shipping labels: \(error)")
+                onCompletion?(error)
+            case .success:
+                onCompletion?(nil)
```

Comment:
Super nit: I normally see the `.success` case first and the `.failure` case later!

Reply:
I actually haven't noticed the common order! sounds good, updated in: https://github.com/woocommerce/woocommerce-ios/pull/3188/commits/f5fff94e8ed20edc38f2f6e4df4f7bb32f85c1f0